api/rust/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 pub use prost;
+pub use tonic;
```

Comment:
Please note that `tonic` is an optional dependency that is behind the `api` feature. Could you change this to:

```rust
#[cfg(feature = "api")]
pub use tonic;
```

Maybe it would be good to also re-export `pbjson_types` (which is behind the `json` feature)?

Reply:
apologies - my notifications from GH got screwed up this last month. thanks for adopting the change!